cypress/e2e/supervision/finance/finance-tab.cy.js

```diff
@@ -0,0 +1,201 @@
+ describe(
+  "Finance tab",
+  { tags: ["@supervision", "@supervision-regression", "@finance-tab", "@finance"] },
+  () => {
+
+    beforeEach(() => {
+      cy.loginAs("Finance User Testing");
+      cy.createClient()
+        .withOrder()
+        .withSupervisionLevel()
+        .withOrderStatus();
+      cy.get("@client").then(({ caseRecNumber }) => {
+        cy.assignSOPNumberToClient(caseRecNumber)
+      });
+      cy.get("@client").then(({ id }) => {
+        cy.visit(`/supervision/#/clients/${id}`);
```

Comment:
You can put this within the `then` above if you include the `id` along with the `caseRecNumber`.